### ISSUE TITLE
Publish blog post to announce Piwik 3 release policy and LTS release schedule

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#10363]
We shall communicate to our users our plans regarding release schedule:

 *  announce Piwik 3 release policy such as new PHP requirements, and availability of first beta version
 * announce  LTS release schedule such as: we only fix security bugs, and major/critical bugs after 2.16.2

**Comment by sgiehl** [comment]
We need to make a feature freeze for Piwik 2.16.x LTS to avoid trouble merging new stuff to Piwik 3.x.
This feature freeze should be after the release of Piwik 2.16.3 and should be announced, so no developer issues feature PRs to LTS afterwards.

**Comment by tsteur** [comment]
The feature freeze was basically when we announced Piwik 2 would be LTS. I finally merged https://github.com/piwik/piwik/pull/10030 which I wanted to do like 3 or 4 months ago already :) Basically all changes should always go into 3.x-dev branch first as LTS should only get serious fixes etc.

**Comment by sgiehl** [comment]
That's right. But we didn't announce that - or at least I haven't seen that. Wasn't included in the LTS announcement. Also we merged quite a few new features for 2.16.2...
So imho we should announce that feature freeze asap, merge all already existing PRs for 2.16.3 (currently 6 or so),  and do not accept any feature PRs afterwards for 2.16.x

Do we already have a plan when to merge 3.x branch to master and moving 2.16.x development to a new branch instead?

**Comment by tsteur** [comment]
> Do we already have a plan when to merge 3.x branch to master and moving 2.16.x development to a new branch instead?

It won't be so soon. Possibly in 2 months or a bit earlier. We could also make Piwik 3 master soon and push master into a `2.x-dev` branch. We should discuss in a week or so I will contact you in chat. There are a few things to consider eg we discussed a few months ago that we might want to use a "develop" branch and only merge "develop" into master when it's green. "develop" would be here "2.x-dev" and "3.x-dev" where we merge "3.x-dev" into master (ideally automatically) when a change was made to 3.x-dev and the build is green.

> That's right. But we didn't announce that - or at least I haven't seen that. Wasn't included in the LTS announcement. Also we merged quite a few new features for 2.16.2...

That's pretty much LTS by definition. We made a few exceptions but don't plan to make more exceptions for this. It's also explained in the blog post what is included: http://piwik.org/blog/2016/01/announcing-long-term-support-in-piwik-2-the-analytics-platform-for-your-mission-critical-projects/
